index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
```

Comment:
Kindly you are including a CSS file named css/style.css, but further down, you are including another CSS file named styles.css. Make sure both files exist and that you are including the appropriate styles for your page.

Reply:
## Changes required
-4/4 done.
-If you need any assistance with this, please let me know and I'll be happy to help.